testing/metadata_array_parsing.sh

```
@@ -1,7 +1,11 @@
 #!/usr/bin/env bash
-
 set -euo pipefail
 
+
+ENABLED=true
```

Shellcheck (0.10.0)

[warning] ENABLED appears unused. Verify use (or export if used externally).

(SC2034)